ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.2.1f1
-m_EditorVersionWithRevision: 2020.2.1f1 (270dd8c3da1c)
+m_EditorVersion: 2020.2.3f1
+m_EditorVersionWithRevision: 2020.2.3f1 (8ff31bc5bf5b)
```

Comment:
Is there a reason for chaning the editor version?

Reply:
@Somafet It wouldn't let me add that version, it only lets you install the latest version:
![image](https://user-images.githubusercontent.com/18538046/107959494-6a5c9b00-6f9b-11eb-85bf-18c9cb6a8af8.png)